ci/gpu/build.sh

```diff
@@ -109,6 +117,5 @@ else
     cat /etc/hosts
 
     gpuci_logger "Python py.test for cuCIM"
-    cd $WORKSPACE/python/cucim
-    py.test --cache-clear -v --ignore-glob . --rootdir=src
+    ./run test python all
```

Comment:
Is this still needed?

Reply:
Yes, it is still needed. I have integrated running test cases from both skimage and clara through `run` script.